libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -45,14 +45,31 @@ func setupIPv6BridgeNetFiltering(config *networkConfiguration, _ *bridgeInterfac
 	return nil
 }
 
+func LoadBridgeModules() error {
```

Comment:
This function doesn't need to be exported (as part of the package interface) ...

```suggestion
func loadBridgeModules() error {
```